src/main/java/com/iggroup/universityworkshopmw/domain/services/ClientService.java

```diff
@@ -50,4 +55,20 @@ Client getClientDataFromMap(String clientId) throws NoAvailableDataException {
          throw new NoAvailableDataException("No available client data in clientIdToClientModelMap for clientId=" + clientId);
       }
    }
+
+   private String checkIfDuplicateClientId() {
+      String uniqueId;
+      do {
+         uniqueId = Helper.createUniqueId(ID_PREFIX);
+      } while (clientIdToClientModelMap.containsKey(uniqueId));
+      return uniqueId;
+   }
+
+   private void checkIfDuplicateUsername(Client client) throws DuplicatedDataException {
+      Optional<Client> duplicatedUsername = clientIdToClientModelMap.values().stream().filter(c -> c.getUserName().equals(client.getUserName())).findFirst();
```

Comment:
could you separate the stream onto multiple lines?

Reply:
yup will do!